x-pack/plugins/data_visualizer/public/application/index_data_visualizer/search_strategy/requests/get_data_view_by_index_pattern.ts

```diff
@@ -18,24 +22,20 @@ import { getESQLAdHocDataview } from '@kbn/esql-utils';
  */
 export async function getOrCreateDataViewByIndexPattern(
   dataViews: DataViewsContract,
-  indexPatternFromQuery: string | undefined,
+  query: string,
   currentDataView: DataView | undefined
 ) {
-  if (indexPatternFromQuery) {
```

Comment:
ℹ️ I am removing this (it checks if there is a persisted dataview). We don't want to associate dataviews with ES|QL so this is not correct. It also aligns with the behavior of the rest apps (Discover, Lens)